portfolio/src/main/webapp/script.js

```diff
@@ -87,43 +84,61 @@ function changePattern(pattern) {
   }
 }
 function createCommentElement(eachComment) {
-    const divElement = document.createElement("div");
-    divElement.className = "col s12 responsive-spaced-box";
 
-    const pElement = document.createElement("p");
-    pElement.className = "z-depth-5 flow-text comment-text";
-    pElement.innerHTML = eachComment.comment;
+  const divElement = document.createElement("div");
+  divElement.className = "col s12 responsive-spaced-box";
 
-    const spanElement = document.createElement("span");
-    spanElement.className = "span-text";
-    spanElement.innerHTML = "By: " + eachComment.name;
+  const pElement = document.createElement("p");
+  pElement.className = "z-depth-5 flow-text comment-text";
+  pElement.innerHTML = eachComment.comment;
 
-    divElement.appendChild(pElement);
-    divElement.appendChild(spanElement);
-    return divElement;
+  const spanElement = document.createElement("span");
+  spanElement.className = "span-text";
+  spanElement.innerHTML =
+    "By: " + eachComment.name + " with email " + eachComment.email;
 
+  divElement.appendChild(pElement);
+  divElement.appendChild(spanElement);
+  return divElement;
 }
 
-async function getCommentCount(){
-    var comment_count = document.getElementById("comment_count").value;
-    const response = await fetch("/data?comment_count=" + comment_count);
-    const message_json = await response.json();
-    
-    var commentHtml = "";
-    var commentContainer = document.getElementById("comment-container");
+async function getCommentCount() {
+  var comment_count = document.getElementById("comment_count").value;
+  const response = await fetch("/data?comment_count=" + comment_count);
+  const message_json = await response.json();
+
+  var commentHtml = "";
+  var commentContainer = document.getElementById("comment-container");
+
+
+  var mainDivElement = document.getElementById("commentContainer");
+  mainDivElement.innerHTML = "";
+
 
-    var mainDivElement = document.getElementById("commentContainer");
-    mainDivElement.innerHTML = "";
+  message_json.forEach(function (comments) {
+    mainDivElement.appendChild(createCommentElement(comments));
+  });
 
-    message_json.forEach(function(comments) {
-        mainDivElement.appendChild(createCommentElement(comments));
-    })
 
-    commentContainer.innerHTML = commentHtml;
+  commentContainer.innerHTML = commentHtml;
 }
 
 function deleteComments() {
-    fetch("/delete-data", {method: "POST"});
-    window.location.reload();
+  fetch("/delete-data", { method: "POST" });
+  window.location.reload();
 }
 
+async function userAuth() {
+  const response = await fetch("/user-auth");
+  const user_details_in_json = await response.json();
+
+  var userStatus = user_details_in_json.userStatus;
+
+  if (userStatus == "true") {
+    document.getElementById("commentForm").style.display = "inherit";
```

Comment:
Rather than directly changing the styles, could you add/remove a `.hide` CSS class? That way all the CSS properties live in the CSS file. Also makes it clear by looking at HTML what is initially hidden and what's not.

Reply:
I've just implemented that.